packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -323,7 +323,7 @@ export const channelsTreeName = ".channels";
 /**
  * Contains telemetry data relevant to summarization workflows.
  * This object is expected to be modified directly by various summarize methods.
- * @public
+ * @alpha
```

Comment:
@markfields @agarwal-navin would appreciate a sanity check to make sure tihs is ok becoming @alpha. `ITelemetryContext` seems to be only related to summarizer flows, but I don't recall how much of summaries are we currently (or want, or are supposed to) exposing as part of @public APIs. The build apparently doesn't complain about this one being alpha so it seems we're not "crossing the @alpha-@public boundary" even after making this change, which I think is a good sign. But still wanted to double check with you if this makes sense conceptually.

Reply:
Seems like never mind, the build is not complaining due to a potential issue with our checks, but it should, and since IChannel is public, ITelemetryContext used for an argument in one of its methods also needs to remain public.

Reply:
@chentong7 let's revert commit [ec69d69](https://github.com/microsoft/FluidFramework/pull/20574/commits/ec69d69dfcde21be090c62be4a7ad0c46ca58e44), ITelemetryContext needs to remain public for now.

Reply:
Done. Thank you, Alejandro.